src/test/groovy/org/shipkit/internal/gradle/init/tasks/InitShipkitFileTest.groovy

```diff
@@ -0,0 +1,70 @@
+package org.shipkit.internal.gradle.init.tasks
+
+import org.junit.Rule
+import org.junit.rules.TemporaryFolder
+import org.shipkit.internal.notes.vcs.GitOriginRepoProvider
+import spock.lang.Specification
+
+class InitShipkitFileTest extends Specification {
+
+    @Rule TemporaryFolder tmp = new TemporaryFolder()
+
+    def "does not modify shipkit file if it already exists"() {
+        given:
+        def shipkitFile = tmp.newFile()
```

Comment:
Is it also more readable (easier for developers reading the code to get know it's `File`)?